src/charactersheet/viewmodels/character/items/view.html

```diff
@@ -0,0 +1,67 @@
+
+<card-edit-actions params="{ flip: $parent.flip }"></card-edit-actions>
+<div data-bind="with: entity">
+    <div class="h4 collapsable-card-title"
+        data-bind="text: name" />
+    <div class="panel-body row">
+    <div class="col-sm-5 col-md-4 col-lg-3">
+        <table class="table table-hover table-condensed table-ac-bordered">
+        <tbody>
+            <tr>
+            <!-- ko if: isContainer -->
+                <th>Carry Weight:</th>
+                <td>
+                    <!-- ko if: contributesToTotalWeight -->
+                    <span style="white-space: nowrap" data-bind="text: totalCalculatedWeightLabel"></span>
+                    <!-- /ko -->
+                    <!-- ko ifnot: contributesToTotalWeight -->
+                        Not Carrying
+                    <!-- /ko -->
+                </td>
+            <!-- /ko -->
+            <!-- ko ifnot: isContainer -->
+                <th>Quantity:</th>
+                <td><span data-bind="text: quantity"></span></td>
+            <!-- /ko -->
+            </tr>
+            <tr>
+            <th>
+                <!-- ko if: isContainer -->
+                Weight (contents):
```

Comment:
I don't remember if I saw these different weights before and they confused me a little.
It took me a bit of trial and error to realize what fields are being represented here.
We have on ` Weight (contents)`, which represents `how much the container itself weights (how much the container itself weights + all of the contents in the container)`
then we have the same thing in `Value`.
Would it be more clear to just split all these fields up into table entries?
You could have:
- Container Weight
- Contents Weight
- Container Value
- Contents Value